src/modules/Dashboard/features/RespondentData/RespondentDataReview/Feedback/FeedbackNotes/FeedbackNotes.tsx

```diff
@@ -37,42 +38,50 @@ export const FeedbackNotes = () => {
     defaultValues: { newNote: '' },
   });
 
-  const { execute } = useAsync(
+  const { execute: getAnswersNotes } = useAsync(
     getAnswersNotesApi,
     (res) => res?.data?.result && setNotes(res.data.result),
   );
 
   const updateListOfNotes = () => {
-    appletId && answerId && execute({ appletId, answerId, params: {} });
+    appletId && answerId && getAnswersNotes({ appletId, answerId, activityId, params: {} });
   };
 
-  const { execute: executeAddNote } = useAsync(createAnswerNoteApi, () => updateListOfNotes());
+  const { execute: createAnswerNote } = useAsync(createAnswerNoteApi, () => updateListOfNotes());
+  const { execute: editAnswerNote } = useAsync(editAnswerNoteApi, () => updateListOfNotes());
+  const { execute: deleteAnswerNote } = useAsync(deleteAnswerNoteApi, () => updateListOfNotes());
 
-  const { execute: executeEditNote } = useAsync(editAnswerNoteApi, () => updateListOfNotes());
-
-  const { execute: executeDeleteNote } = useAsync(deleteAnswerNoteApi, () => updateListOfNotes());
-
-  useEffect(() => {
-    if (appletId && answerId) {
-      execute({ appletId, answerId, params: {} });
-    }
-  }, [appletId, answerId]);
+  const activityId = activity?.id ?? '';
 
   const handleNoteEdit = (updatedNote: Pick<FeedbackNoteType, 'id' | 'note'>) => {
     appletId &&
       answerId &&
-      executeEditNote({ appletId, answerId, noteId: updatedNote.id, note: updatedNote.note });
+      editAnswerNote({
+        appletId,
+        answerId,
+        activityId,
+        noteId: updatedNote.id,
+        note: updatedNote.note,
+      });
   };
 
   const handleNoteDelete = (noteId: string) => {
-    appletId && answerId && executeDeleteNote({ appletId, answerId, noteId });
+    appletId && answerId && deleteAnswerNote({ appletId, answerId, activityId, noteId });
   };
 
   const addNewNote = () => {
-    appletId && answerId && executeAddNote({ appletId, answerId, note: getValues().newNote });
+    appletId &&
+      answerId &&
+      createAnswerNote({ appletId, answerId, activityId, note: getValues().newNote });
     setValue('newNote', '');
   };
 
+  useEffect(() => {
+    if (appletId && answerId) {
```

Comment:
appletId && answerId && getAnswersNotes({ appletId, answerId, activityId, params: {} });
```suggestion
    if (appletId && answerId) {
```